cmd/ingester/app/flags.go

```diff
@@ -64,6 +68,7 @@ const (
 // Options stores the configuration options for the Ingester
 type Options struct {
 	kafkaConsumer.Configuration
+	ClientID         string
```

Comment:
I didn't notice this earlier - but we should move this to `kafkaConsumer.Configuration` which is where GroupID/Topic/etc are configured.

Reply:
yes it is redundant. it is removed.